module/Activity/src/Activity/Form/Activity.php

```diff
@@ -16,13 +21,26 @@ class Activity extends Form implements InputFilterProviderInterface
     protected $inputFilter;
     protected $organs;
 
-    public function __construct()
+    /**
+     * @param Organ[] $organs
+     * @param Translator $translator
+     */
+    public function __construct(array $organs, Translator $translator)
     {
         parent::__construct('activity');
         $this->setAttribute('method', 'post');
         $this->setHydrator(new ClassMethodsHydrator(false))
             ->setObject(new \Activity\Model\Activity());
 
+        // all the organs that the user belongs to in organId => name pairs
+        $organOptions = [0 => $translator->translate('No organ')];
```

Comment:
It is superfluous from a parsing perspective, it is not superfluous from an understanding perspective. Since it is a map, and not an array, the key should be provided for understandability